Makefile

```diff
@@ -45,3 +45,18 @@ test:
 
 testcov:
 	py.test --cov=ideascube/ --cov-report=term-missing --migrations
+
+test-data-migration:
+	set -e ; \
+	BRANCH=$$(git rev-parse --abbrev-ref HEAD); \
+	LATEST_TAG=$$(git describe --abbrev=0 --tags); \
+	\
+	echo "# Loading some data at $$LATEST_TAG"; \
+	git checkout $$LATEST_TAG; \
+	rm -fr storage; \
+	make migrate; \
+	python manage.py loaddata --database=default test-data/data.json; \
```

Comment:
We agreed on IRC that using this as an occasion to test the reindexation process was a good idea, as it only adds ~4 seconds to the job. (it's almost a factor 2, though)